### PR TITLE
Add citation info

### PR DESCRIPTION
**Pull request by Peter9192**
and licences, ....

### CODE REVIEW

**Review by sverhoeven — COMMENTED** [review]
I corrected the repository code URL.

CFF is valid according to cffconvert.

**Review by sverhoeven — COMMENTED** [review]
I have added a link from README to the cff file.

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  -
+    family-names: Palmer
+    given-names: Tamzin
+    orcid: "https://orcid.org/0000-0002-8889-5595"
+    affiliation: UK Met Office
+  -
+    family-names: Ribes
+    given-names: Aurélien
+    affiliation: Centre National de Recherches Météorologiques
+    orcid: 0000-0001-5102-7885
```
Comment (SarahAlidoost) [inline CITATION.cff:73]:
```suggestion
    orcid: https://orcid.org/0000-0001-5102-7885
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# YAML 1.2
+# More then one affiliation:
```
Comment (SarahAlidoost) [inline CITATION.cff:2]:
```suggestion
# More than one affiliation:
```

File: CITATION.cff (hunk 1 above)
Comment (bvreede) [reply CITATION.cff:73]: + add quotes to this one, like done to the others!

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  -
+    family-names: Palmer
+    given-names: Tamzin
+    orcid: "https://orcid.org/0000-0002-8889-5595"
+    affiliation: UK Met Office
+  -
+    family-names: Ribes
+    given-names: Aurélien
+    affiliation: Centre National de Recherches Météorologiques
+    orcid: https://orcid.org/0000-0001-5102-7885
```
Comment (SarahAlidoost) [inline CITATION.cff:73]:
```suggestion
    orcid: "https://orcid.org/0000-0001-5102-7885"
```

**Review by SarahAlidoost — APPROVED** [review]
@Peter9192 thanks.

**Pull request merged**